lib/awful/prompt.lua

```diff
@@ -210,8 +209,9 @@ end
 -- @param id The data.history identifier
 local function history_save(id)
     if data.history[id] then
-        assert(gfs.make_parent_directories(id))
-        local f = assert(io.open(id, "w"))
+        gfs.make_parent_directories(id)
+        local f = io.open(id, "w")
+        if not f then return end
```

Comment:
shouldn't it at least write some warning instead of just skipping the error silently?